packages/desktop-electron/index.ts

```diff
@@ -327,15 +353,15 @@ ipcMain.on('screenshot', () => {
   }
 });
 
-ipcMain.on('update-menu', (event, budgetId?: string) => {
+ipcMain.on('update-menu', (_event, budgetId?: string) => {
   updateMenu(budgetId);
 });
 
-ipcMain.on('set-theme', theme => {
+ipcMain.on('set-theme', (_event, theme: string) => {
   const obj = { theme };
   if (clientWin) {
     clientWin.webContents.executeJavaScript(
-      `window.__actionsForMenu && window.__actionsForMenu.saveGlobalPrefs(${obj})`,
+      `window.__actionsForMenu && window.__actionsForMenu.saveGlobalPrefs(${JSON.stringify(obj)})`,
```

Comment:
This is a bug - it would only be seen when triggering the theme from the dev console using: 

```Actual.setTheme('light')```

My assumption is that has never worked - it was sending coming through as [Object object]